src/wasm-stack.h

```diff
@@ -493,8 +498,10 @@ class StackIRToBinaryWriter {
 public:
   StackIRToBinaryWriter(WasmBinaryWriter& parent,
                         BufferWithRandomAccess& o,
-                        Function* func)
-    : writer(parent, o, func, false /* sourceMap */, false /* DWARF */),
+                        Function* func,
+                        bool memory64)
+    : writer(
+        parent, o, func, false /* sourceMap */, false /* DWARF */, memory64),
```

Comment:
the convention (in chromium anyway IIRC) is to do `/*sourceMap=*/false` .. with no spaces.